src/jit/ee_il_dll.cpp

```diff
@@ -439,15 +439,39 @@ unsigned           Compiler::eeGetArgSize(CORINFO_ARG_LIST_HANDLE list, CORINFO_
         assert(argTypeJit != CORINFO_TYPE_REFANY || structSize == 2*sizeof(void*));
 
 #if FEATURE_MULTIREG_ARGS
-#ifdef _TARGET_ARM64_
+        // For each target that supports passing struct args in multiple registers 
+        // apply the target specific rules for them here:
+#if defined(_TARGET_ARM64_)
+        // For ARM64 any structs that are larger that 32 bytes are passed by reference
```

Comment:
Sorry I was thinking of the integer case.  Wasn't clear to me if this was the max int or max of fp/int.

For this method do we see that there's a per platform ifdef?  How about for ARM chips with out FP units? (maybe a rare case but Samsung might care) I can see these functions getting very unwieldy.